text/0497-glue-l2-construct.md

```diff
@@ -7,57 +7,77 @@
-[L1 (CloudFormation) Constructs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_Glue.html) building on the
-functionality already supported in the [@aws-cdk/aws-glue-alpha module](https://github.com/aws/aws-cdk/blob/v2.51.1/packages/%40aws-cdk/aws-glue/README.md).
+development.
+
+Wihout an L2 construct, developers define Glue data sources, connections,
+jobs, and workflows for their data and ETL solutions via the AWS console,
+the AWS CLI, and Infrastructure as Code tools like CloudFormation and the
+CDK. However, there are several challenges to defining Glue resources at
+scale that an L2 construct can resolve. First, developers must reference
+documentation to determine the valid combinations of job type, Glue version,
+worker type, language versions, and other parameters that are required for specific
+job types. Additionally, developers must already know or look up the
+networking constraints for data source connections, and there is ambiguity
+around how to securely store secrets for JDBC connections. Finally,
+developers want prescriptive guidance via best practice defaults for
+throughput parameters like number of workers and batching.
+
+The Glue L2 construct has convenience methods working backwards from common
+use cases and sets required parameters to defaults that align with recommended
+best practices for each job type. It also provides customers with a balance
+between flexibility via optional parameter overrides, and opinionated
+interfaces that discouraging anti-patterns, resulting in reduced time to develop
+and deploy new resources.
+
+### References
+
+* [Glue Launch Announcement](https://aws.amazon.com/blogs/aws/launch-aws-glue-now-generally-available/)
+* [Glue Documentation](https://docs.aws.amazon.com/glue/index.html)
+* [Glue L1 (CloudFormation) Constructs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_Glue.html)
+* Prior version of the [@aws-cdk/aws-glue-alpha module](https://github.com/aws/aws-cdk/blob/v2.51.1/packages/%40aws-cdk/aws-glue/README.md)
 
 ## Create a Glue Job
 
-The glue-alpha-module already supports three of the four common types of Glue
-Jobs: Spark (ETL and Streaming), Python Shell, Ray. This RFC will add the
-more recent Flex Job. The construct also implements AWS best practice
-recommendations, such as:
-
-* use of Secrets Management for Connection JDBC strings
-* Glue Job Autoscaling
-* defaults for Glue job specification
-
-This RFC will introduce breaking changes to the existing glue-alpha-module to
-streamline the developer experience and introduce new constants and validations.
-As an opinionated construct, the Glue L2 construct will enforce
-best practices and not allow developers to create resources that use deprecated
-libraries and tool sets (e.g. deprecated versions of Python).
-
-Optional and required parameters for each job will be enforced via interface
+A Job encapsulates a script that connects to data sources, processes
```

Comment:
Nice! When you go and write the actual readme in aws-cdk-lib, the first paragraph is the perfect intro + an example. The second paragraph (starting with "This iteraton of the L2 construct") is less readme-ey. We'll call out the breaking changes in the PR description so they show up in the changelog, and customers better know that L2 === opinionated well before reading the readme.